packages/d/discover/package.yml

```diff
@@ -0,0 +1,47 @@
+name       : discover
+version    : 5.27.10
+release    : 1
+source     :
+    - https://cdn.download.kde.org/stable/plasma/5.27.10/discover-5.27.10.tar.xz : 1f9189e2a759ac035a5ba83a2842a40b7034a7e5885b0836e3921b9671109fa8
+homepage   : https://apps.kde.org/discover/
+license    : GPL-2.0-or-later
+component  : desktop.kde
```

Comment:
I think `desktop.kde` is automatically installed to the ISOs, should we use a different one temporarily so this package isn't auto-installed?

Reply:
'@desktop.kde.core' we should be good